tools/mason/MasonHelp.chpl

```diff
@@ -132,6 +134,24 @@ proc masonSearchHelp() {
   writeln(desc);
 }
 
+proc masonModifyHelp() {
+  writeln("Modify a Mason package's Mason.toml");
+  writeln();
+  writeln("Usage:");
+  writeln("    mason rm [options] package");
+  writeln("    mason add [options] package@version");
```

Comment:
Just noticed this with `mason {add,rm}`, but it's an issue across many commands: `mason <subcommand> --help` outside of a mason project results in:

```
Mason could not find your configuration file (Mason.toml)
```

Users should definitely be able to view help output outside of a mason project.

If this turns out to require a significant change to fix, I'd be OK with documenting this an issue and taking it 
 in separate this PR.